src/main/java/dev/hephaestus/glowcase/block/entity/ItemDisplayBlockEntity.java

```diff
@@ -40,8 +38,12 @@ public class ItemDisplayBlockEntity extends BlockEntity implements Inventory {
 	public float yaw;
 	public Set<UUID> givenTo = new HashSet<>();
 
+	public ItemDisplayBlockEntity(BlockEntityType<?> type, BlockPos pos, BlockState state) {
+		super(type, pos, state);
+	}
+
 	public ItemDisplayBlockEntity(BlockPos pos, BlockState state) {
-		super(Glowcase.ITEM_DISPLAY_BLOCK_ENTITY.get(), pos, state);
+		this(Glowcase.ITEM_DISPLAY_BLOCK_ENTITY.get(), pos, state);
```

Comment:
What is the reason behind this change?

Reply:
No. This was part of an attempt to abstractify the class for the entity display to reduce copy-pasting before I decided against that. Forgot to change this back though